.pre-commit-config.yaml

```diff
@@ -34,6 +34,11 @@ repos:
     hooks:
     -   id: check-github-actions
     -   id: check-github-workflows
+-   repo: https://github.com/shellcheck-py/shellcheck-py
+    rev: v0.7.1.1
+    hooks:
+      - id: shellcheck
+        files: '^.*.sh$'
```

Comment:
I'd use `types` instead. See https://pre-commit.com/#filtering-files-with-types. You'll also get like, extensionless files but with the shebang. I ran `identify-cli` on some random executable extensionless shell script:

`["bash", "executable", "file", "shell", "text"]`

I'd use the `shell` type.

Reply:
That's neat! Thanks for the suggestion.